### PR TITLE
Thibault/documentation

### PR DESCRIPTION
**Pull request by thibmonsel**
Main page for documentation maybe need to import latex into oxygen.

### CODE REVIEW

File: doc/pages/MAINPAGE.md [hunk 1]
```diff
@@ -4,4 +4,95 @@ To get started on the project, all the instructions are linked into the README:
 @subpage readme
 
 To get started on Olivier's thesis, here is the documentation related to our implementation of the metaheuristic:
-@subpage learning_algorithms
\ No newline at end of file
+@subpage learning_algorithms
+
+# Fast Preference Learning
```
Comment (Mostah) [inline doc/pages/MAINPAGE.md:9]: Markdown wise I would recommend having only one title at `#` level, I also find it a bit more esthetic to have a big title and sub titles for the rest. However it's up to you for the change ;)

File: doc/pages/MAINPAGE.md [hunk 2]
```diff
@@ -4,4 +4,95 @@ To get started on the project, all the instructions are linked into the README:
...
+
+*A **criterion** is a function $$g_j$$ assigning a value to each alternative of a decision problem such that it becomes possible for a DM to perform pairwise comparisons and express a preference in favor of one alternative based on this value.*
+
+The Cartesian product of criteria scales is denoted by  X = $$\prod_{j=1}^{n}$$ $$X_j$$ in which $$X_j$$ represents all the possible values of a criterion j. 
+We designate $$g_j$$ : $$A$$ $$\to$$ $$X_j$$ the function assigning a value to each alternative of the set $$A$$. $$A$$ represents a finite set of alternatives for which a ranking or a sorting has to be determined. Let $$a$$ $$\in$$ $$A$$ be an alternative $$a$$ contained in the set $$A$$. We denote by $$a_j$$ = $$g_j(a)$$ the performance of an alternative a on criterion $$j$$. The performance vector associated to each alternative a is denoted by $$a$$ = ($$a_1$$, ..., $$a_n$$);
+
+There exist different **types of preference relation** between two alternatives. Roy and Bouyssou (1993) list four types of preferences:
```
Comment (Mostah) [inline doc/pages/MAINPAGE.md:42]: Should we talk about that here? As we don't need it for our algorithm

**Review by Mostah — APPROVED** [review]

File: doc/pages/MAINPAGE.md (hunk 2 above)
Comment (thibmonsel) [reply doc/pages/MAINPAGE.md:42]: Doesn't it give general information on Mcda models ?

File: doc/pages/MAINPAGE.md (hunk 1 above)
Comment (thibmonsel) [reply doc/pages/MAINPAGE.md:9]: yes defintely

**Pull request merged**